### PR TITLE
Feature/select component

### PR DESCRIPTION
**Pull request by DominikGuzei**
This PR adds the first basic version of the `<Select>` component which can be used as drop-down or drop-up depending on your needs (does not have any auto logic based on screen state).

**NORMAL:**
screenshot 2017-05-04 um 12 00 54

**UPWARD:**
screenshot 2017-05-04 um 12 07 47

### CODE REVIEW

File: source/themes/simple/SimpleSelect.scss [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+.options {
+  position: relative;
+  font-size: 16px;
+  font-family: $theme-font-light;
+  border: 1px solid #c6cdd6;
+  list-style: none;
+  padding: 0;
+  border-radius: 2px;
```
Comment (nikolaglumac) [inline source/themes/simple/SimpleSelect.scss]: @DominikGuzei I would add `box-sizing: border-box;` here as currently options dropdown is 2px wider than the select field (due to width:100% + border combo).

**Comment by nikolaglumac** [comment]
@DominikGuzei select box looks great! 👍 
What do you think about adding two keyboard event interactions:
- close opened dropdown on ESC key
- browse through dropdown options using up/down arrow keys (plus selection with ENTER key)?

**Comment by nikolaglumac** [comment]
@DominikGuzei I have added keyboard interaction (closing open dropdown on ESC key, changing selection with Arrows Up/Down and selecting highlighted option using ENTER key).
Besides that I have improved styling and fixed few bugs (e.g. it was not possible to open dropdown if you click on small arrow on the right side of the input).

**Comment by nikolaglumac** [comment]
@DominikGuzei I have added keyboard interaction (closing open dropdown on ESC key, changing selection with Arrows Up/Down and selecting highlighted option using ENTER key).
Besides that I have improved styling and fixed few bugs (e.g. it was not possible to open dropdown if you click on small arrow on the right side of the input).

**Comment by nikolaglumac** [comment]
In latest commit I have added support for FormField label click-to-focus feature.
This feature on input/textarea elements add's focus and on select-fields it opens dropdown.

**Comment by nikolaglumac** [comment]
@DominikGuzei I have added `optionRenderer` option for the select box ;)

## Normal options:
screen shot 2017-05-11 at 11 59 23

## Custom options:
screen shot 2017-05-11 at 14 00 17

**Comment by DominikGuzei** [comment]
@nikolaglumac pure awesomeness! 😃

**Pull request merged**